chat-monitor.user.js

```diff
@@ -14,10 +15,15 @@
 // @grant       GM_getValue
 // @grant       GM_setValue
 // @grant       GM_log
-// @resource style https://raw.githubusercontent.com/paul-lrr/nifty-chat-monitor/master/chat-monitor.css
+// @resource style https://raw.githubusercontent.com/paul-lrr/nifty-chat-monitor/new-twitch/chat-monitor.css
```

Comment:
Would it be possible to use a relative path for the stylesheet? Having an absolute URL with a specific git branch name hardcoded in sort of defeats the ability to use git branches at all

Reply:
As far as I'm aware it's not possible to use relative paths as those would be relative to the website they're running on. The only alternative that I know of (that I usually apply, but I didn't want to throw everything overboard here) is having the script add the CSS style itself, like so:

    var styleElement = document.createElement('style');
    styleElement.textContent = `# CSS stylesheet goes here
    # it can even span multiple lines with the backtick notation
    body { color: blue }`;
    document.head.appendChild(styleElement);